app/src/main/java/org/jellyfin/mobile/utils/ApiExtensions.kt

```diff
@@ -33,6 +34,10 @@ suspend fun ApiClient.markPlayed(itemId: String, userId: String): UserItemDataDt
     MarkPlayedAsync(itemId, userId, Date(), ContinuationResponse(continuation))
 }
 
+suspend fun ApiClient.getServerConfigurationAsync(): ServerConfiguration? = suspendCoroutine { continuation ->
```

Comment:
To be more in line with the other functions:
```suggestion
suspend fun ApiClient.getServerConfiguration(): ServerConfiguration? = suspendCoroutine { continuation ->
```

Also, you can move this below `getPublicSystemInfo`.